### ISSUE TITLE
SSHKey didn't seem to take following the Getting Started Guide

### ISSUE THREAD

**Issue by slovefatpot** [kelseyhightower/coreos-ipxe-server#8]
Hey,

  I'm running an iPXE server instance on a Windows 7 64-bit desktop.  I'm iPXE-booting a VirtualBox instance.  I worked through the "Getting Started Guide" and was successful getting the VirtualBox instance booted into CoreOS.

  When I tried to ssh (using PuTTY) from the desktop to the successfully booted CoreOS VirtualBox instance, I kept getting a password prompt.  The way I finally fixed the issue was to add a line to the 'development.json' file:

```
{
    "cloud_config": "development",
    "rootfstype": "btrfs",
    "sshkey": "coreos", // this is the line I added
    "version": "310.1.0"
}
```

  Once that line was there, I was able to connect to the CoreOS instance.  It seems that in my case, the sshkey command-line parameter to the kernel wasn't defaulting to the 'coreos' key . . . no idea why . . .

  Otherwise, this is a really great project!  Thanks for your efforts!

**Comment by kelseyhightower** [comment]
I'll get that fixed up. Thanks for the bug report!

**Comment by kelseyhightower** [comment]
Fixed in 395c74658a208de58033bcc62eef5a0d7d798c55
